migrations/20200507120000-create-comment-reactions.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    await queryInterface.createTable('CommentReactions', {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      UserId: {
```

Comment:
I just realized that I forgot to add a column to reference the `FromCollective` in the specs:

```suggestion
      FromCollectiveId: {
        type: DataTypes.INTEGER,
        references: { key: 'id', model: 'Users' },
        onDelete: 'CASCADE',
        onUpdate: 'CASCADE',
        allowNull: false,
      },
      UserId: {
```